lib/resources/r4/related_person.yaml

```diff
@@ -75,4 +76,313 @@ fields:
     terminology:
       - display: Relationship Level
         system: http://hl7.org/fhir/resource-types
-        info_link: https://www.hl7.org/fhir/valueset-resource-types.html
\ No newline at end of file
+        info_link: https://www.hl7.org/fhir/valueset-resource-types.html
+
+- name: resourceType
+  required: 'Yes'
+  type: string
+  description: The type of the FHIR resource.
+  action: create
+  example: |
+    {
+      "resourceType": "RelatedPerson"
+    }
+  note: resourceType must be RelatedPerson.
+  url: http://hl7.org/fhir/R4/json.html#resources
+
+- name: relationship
+  required: 'Yes'
+  type: CodeableConcept
+  action: create
+  description: The nature of the relationship between a patient and the related person.
+  example: |
+    {
+      "relationship": [
+        {
+          "coding": [
+            {
+              "system": "https://fhir.cerner.com/a758f80e-aa74-4118-80aa-98cc75846c76/codeSet/351",
+              "code": "1152",
+              "display": "child"
+            }
+          ]
+        }
+      ]
+    }
+  note: |
+    <ul>
+      <li>The <code>relationship.coding</code> must contain exactly one proprietary system.</li>
+      <li>The <code>relationship.coding</code> cannot contain a standard system.</li>
+    </ul>
+
+- name: patient
+  required: 'Yes'
+  type: Reference (Patient)
+  action: create
+  description: Patient that the individual is related to for the encounter.
```

Comment:
'for the encounter' doesn't make sense here. Maybe this should be 'for the RelatedPerson'?

Reply:
or for the 'Encounter-level RelatedPerson'

Reply:
updated here:https://github.com/cerner/fhir.cerner.com/pull/527/commits/91acf8cf0ff17eb8973b5b12811428fec2b129c6#diff-261586474bd9faacdccf38c13eea61175a7bba5b7ce8ddb4dea273142eb9f6d6R122